server/controllers/componentsController.ts

```diff
@@ -122,11 +88,16 @@ export default ({
           [componentName]: viewModels[index],
         }
       },
-      { meta: userData },
+      {
+        meta:
+          user.authSource === 'nomis'
+            ? { caseLoads: user.caseLoads, activeCaseLoad: user.activeCaseLoad, services: user.services }
+            : DEFAULT_USER_ACCESS,
```

Comment:
Preserving the behaviour for non prison users where the `meta` would contain empty case load lists and services